model/atmosphere/advection/src/icon4py/model/atmosphere/advection/stencils/compute_antidiffusive_cell_fluxes_and_min_max.py

```diff
@@ -0,0 +1,107 @@
+# ICON4Py - ICON inspired code in Python and GT4Py
+#
+# Copyright (c) 2022-2024, ETH Zurich and MeteoSwiss
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import gt4py.next as gtx
+from gt4py.next.ffront.fbuiltins import (
+    astype,
+    broadcast,
+    maximum,
+    minimum,
+    neighbor_sum,
+)
+
+from icon4py.model.common import dimension as dims, field_type_aliases as fa, type_alias as ta
+from icon4py.model.common.dimension import C2CE, C2E
+from icon4py.model.common.settings import backend
+from icon4py.model.common.type_alias import vpfloat
+
+
+@gtx.field_operator
+def _compute_antidiffusive_cell_fluxes_and_min_max(
+    geofac_div: gtx.Field[gtx.Dims[dims.CEDim], ta.wpfloat],
+    p_rhodz_now: fa.CellKField[ta.wpfloat],
+    p_rhodz_new: fa.CellKField[ta.wpfloat],
+    z_mflx_low: fa.EdgeKField[ta.wpfloat],
+    z_anti: fa.EdgeKField[ta.wpfloat],
+    p_cc: fa.CellKField[ta.wpfloat],
+    p_dtime: ta.wpfloat,
+) -> tuple[
+    fa.CellKField[ta.vpfloat],
+    fa.CellKField[ta.vpfloat],
+    fa.CellKField[ta.wpfloat],
+    fa.CellKField[ta.vpfloat],
+    fa.CellKField[ta.vpfloat],
+]:
+    zero = broadcast(vpfloat(0.0), (dims.CellDim, dims.KDim))
```

Comment:
same here, this should not pass
```suggestion
    zero = broadcast(0.0, (CellDim, KDim))
```
Also you don't need to cast `vpfloat` on a singular float